WreckEngine/GLError.h

```diff
@@ -14,40 +18,50 @@
 #define CLEAR_GL_ERR glGetError()
 #define CHECK_GL_ERR GLCheckError("GL-ERROR; \"" __FILE__ "\" before line " STRINGIZE(__LINE__) ": \n")
 
+#if defined(_DEBUG) && !defined(NDEBUG)
+#define GL_CHECK(x) { x; ::CheckGlErrorImpl(#x, __FILE__, __LINE__, __PRETTY_FUNCTION__); }
```

Comment:
`__PRETTY_FUNCTION__` is defined in non-MSVC compilers. If you look a couple of lines up, you'll see I define it in the case `_MSC_VER` is defined. I'll change it to be compiler agnostic, but like we discussed I wrote that code very quickly to track down a GL error.